root/components/relationship-editor.tt

```diff
@@ -47,11 +47,36 @@
     </div>
     <div>
       <label>
-        <input type="checkbox" data-bind="checked: $parent.changeAllRelationshipCredits" />
-        <span data-bind="html: MB.i18n.l('Change all relationship credits for {entity} on the page.', {entity: entity.html({target: '_blank'})})">
+        <input type="checkbox" data-bind="checked: $parent.changeOtherRelationshipCredits[role]" />
+        <span data-bind="html: MB.i18n.l('Change credits for other {entity} relationships on the page.', {entity: entity.html()})">
         </span>
       </label>
     </div>
+    <div data-bind="visible: $parent.changeOtherRelationshipCredits[role]">
+      <div>
+        <label>
+          <input type="radio" value="all" data-bind="checked: $parent.selectedRelationshipCredits[role]" />
+          [% l('All of these relationships.'); %]
+        </label>
+      </div>
+      <div>
+        <label>
+          <input type="radio" value="same-entity-types" data-bind="checked: $parent.selectedRelationshipCredits[role]" />
+          <span data-bind="html: MB.i18n.l('Only relationships to {entity_type} entities.',
+                                           {entity_type: MB.i18n.strings.entityName[relationship.target(entity).entityType]})">
+          </span>
+        </label>
+      </div>
+      <div>
+        <label>
+          <input type="radio" value="same-relationship-type" data-bind="checked: $parent.selectedRelationshipCredits[role]" />
+          <span data-bind="html: MB.i18n.l('Only “{relationship_type}” relationships to {entity_type} entities.',
+                                           {relationship_type: $parent.linkTypeName(),
+                                            entity_type: MB.i18n.strings.entityName[relationship.target(entity).entityType]})">
```

Comment:
Ideally the entity type wouldn't be capitalized in the resulting string, but it's hard to fix:

1. I don't think we want to make people translate lowercase versions of every string.
2. `String.prototype.toLocaleLowerCase()` uses the locale of the browser, not of MusicBrainz. Maybe that's better than nothing.

I guess this is something we can figure out later, after this is merged.